spec/requests/api/app_data_index_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe 'GET /api/app_data' do
+  let!(:api_key) { Rails.application.credentials.client_api_keys[0] }
+
+  describe 'with a valid api key' do
+    before do
+      get '/api/app_data', headers: { 'API_KEY' => api_key }
```

Comment:
It is out of scope now. But is this key different from one on the client user? And do you plan to differentiate admin requests by keys or Devise headers?
Admin can request info that is not available to ordinary users.

Reply:
For now this is a request made by the client. In order to populate the app with data. Just the api key is needed for that. Update action is not in scope, but yes. We will require authentication